i18n/ja.yaml

```diff
@@ -4,6 +4,18 @@ toggleMenu:
 darkMode:
     other: ダークモード
 
+list:
+    page:
+        one: "{{ .Count }} ページ目"
+        other: "{{ .Count }} ページ目"
+
```

Comment:
They can be merges into one entry since they are the same I believe.

```yaml
list:
    page:
        other: "{{ .Count }} ページ目"
```

Reply:
@CaiJimmy Applied the suggestion.